CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Current develop
 
 ### Added (new features/APIs/variables/...)
+- [[PR 588]](https://github.com/lanl/parthenon/pull/588) Switch from nbmax to nneighbors in particles
```

Comment:
This should probably go to the "fixed" section.

Reply:
Agreed, thanks (actually never noticed the other `CHANGELOG` sections, oops)